src/components/generic/Table/TableRowItem/TableRowItem.js

```diff
@@ -20,19 +23,53 @@ const TableRowItem = ({
   isOriginalSelected,
   isDuplicateSelected,
   isAllowNewlines,
+  isLink,
 }) => {
   const rowItemValue = options ? getOptionsByItemLabelOrName(value, options) : value
   const extraRowItemValue = options ? getOptionsByItemLabelOrName(extraValue, options) : extraValue
   const hasExtraRowForDuplicateRecord = extraValue !== undefined
   const highlightedCurrentSite = isOriginalSelected ? 'highlighted' : undefined
   const highlightedDuplicateSite = isDuplicateSelected ? 'highlighted' : undefined
 
+  const navigate = useNavigate()
```

Comment:
Can we make the buttons links instead of buttons? So that a user has the context menu for a link (and can choose to open up a link in a new tab)?

<img width="1045" alt="Screenshot 2024-01-17 at 11 50 08 AM" src="https://github.com/data-mermaid/mermaid-webapp/assets/1740152/35238c64-ba6e-4974-8466-68ae1076a1a0">
<img width="1045" alt="Screenshot 2024-01-17 at 11 49 23 AM" src="https://github.com/data-mermaid/mermaid-webapp/assets/1740152/ee890e12-2d67-42d4-9d69-2a26770bf739">

Reply:
yup good call! I also prefer it to open in a new tab